src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataElasticServiceImpl.java

```diff
@@ -45,7 +44,7 @@ public void saveOrUpdateToElastic(FormDataDefinition formDataDefinition)
     {
         if(elasticEnable) {
             try {
-                webClientWrapper.webclientRequest(webClientWrapper.createWebClient(tokenUtils.getTokenFromContext()),
+                webClientWrapper.webclientRequest(webClientWrapper.createWebClient("Bearer "+tokenUtils.getTokenFromContext()),
```

Comment:
you sure we need this change

Reply:
Yes vaibhav its required to Bearer because its going into exception otherwise